### PR TITLE
Update readme with config info

### PR DESCRIPTION
**Pull request by neomatrix369**
DO NOT MERGE till PR #21 is merged

Contains configuration information on how to go about with setting up the Machine User and using the token generated from it to setup the Meterian Jenkins Plugin.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,6 +41,31 @@ This kicks off a full-fledged Jenkins instance and can be access via http://0.0.
...
+- `target/meterian-plugin.jar`
+
+The `meterian-plugin.hpi` is the artifact that will be installed as a Jenkins plugin, via the Jenkins, see https://jenkins.io/doc/book/managing/plugins/.
+
+## Configuration
+
+In order to run the tests locally we would need the following two environment variables populated with valid values or else the tests will fail:
+
+- `METERIAN_API_TOKEN` - this is generated via the Meterian dashboard (via https://www.meterian.com, you will need an account) 
+- `METERIAN_GITHUB_TOKEN` - this is the personal token generated from the machine user account created by yourself (learn more about machine user creation at [GitHub: developer guides | machine-users](https://developer.github.com/v3/guides/managing-deploy-keys/#machine-users)), follow the steps mentioned in the link
```
Comment (bbossola) [inline README.md]: I would probably call this, here and in the code, simply "GITHUB_TOKEN" as it's related to GITHUB, not to Meterian

Comment (bbossola) [reply README.md]: Also added to Circle-CI

Comment (neomatrix369) [reply README.md]: I'll beg to differ my response to this, as by having generically named environment variables we might open up opportunities for mistakes or bugs (reading into or writing to incorrect unindented variables of the user or other apps). Its a good practise keeping Meterian plugin related config to be distinct and distinguishable for the end-user. 

For the same reasons, I find CircleCI (https://circleci.com/docs/2.0/env-vars/#built-in-environment-variables) and Dockers (https://docs.docker.com/v17.09/engine/reference/commandline/cli/#environment-variables) environment variable names prefixed respectively, easy to read and remember. 

I had a similar experience on my end when looking at my local machine config file, and environment variables so decided to prefix them to able to separate them from my other variables.

Security/Sys/DevOps admins might choose to create separate tokens and user names just for Meterian plugin for various reasons.

I can keep the names as you suggest but I see a downside to it.

Comment (neomatrix369) [reply README.md]: > Also added to Circle-CI

Thanx for doing this, alltough the PR is still showing your user id as the end-user who created the PR, see https://github.com/MeterianHQ/autofix-sample-maven-upgrade/pull/54.

I'll re-run fresh builds to see if this changes.

**Comment by neomatrix369** [comment]
Rebase done! No code changes just Docs.

File: README.md (hunk 1 above)
Comment (bbossola) [reply README.md]: Okay, as soon as we prefix everything with METERIAN **but** we keep the name of the services, I am fine.

So I propose:

METERIAN_GITHUB_TOKEN (kept as it is now)
METERIAN_GITHUB_USER 
METERIAN_GITHUB_EMAIL

Comment (neomatrix369) [reply README.md]: Ok i'll rename them in the docs and in the code as well

**Comment by neomatrix369** [comment]
@bbossola this PR should be good to go have a look at the new changes!

**Pull request merged**